### PR TITLE
Add `rel="noopener noreferrer"` in documentation links

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description
the `rel="noopener noreferrer"` attribute is added to external links
 
### Issues Resolved

- #5038 

### Evidence

image
image
image

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 
 ### Added
 
--
+- Add rel="noopener noreferrer" in documentation links. [#5197](https://github.com/wazuh/wazuh-kibana-app/pull/5197)
```
Comment (AlexRuiz7) [inline CHANGELOG.md:9]: ChangeLog entries must always use past tense (Add → Added).

**Review by AlexRuiz7 — CHANGES_REQUESTED** [review]

**Comment by Desvelao** [comment]
CodeQL scan: https://github.com/wazuh/wazuh-kibana-app/actions/runs/4222981621

**Comment by Desvelao** [comment]
# Wazuh dashboard

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check the value of attributes in changed anchors containing noopener noreferer | :green_circle: | :black_circle: | :black_circle: |

**Details**
<details>
<summary>:green_circle: Check the value of attributes in changed anchors containing noopener noreferer</summary>

Chrome - :green_circle:
image
image
image
image
Firefox - :black_circle:

Safari - :black_circle:

</details>

**Review by Desvelao — APPROVED** [review]
**review**

:green_circle: code
:green_circle: test

**Review by AlexRuiz7 — APPROVED** [review]

**Pull request merged**